kairos-cli/src/commands/deposit.rs

```diff
@@ -0,0 +1,26 @@
+use crate::common::args::{AmountArg, PrivateKeyPathArg};
+use crate::crypto::error::CryptoError;
+use crate::crypto::signer::CasperSigner;
```

Comment:
Why did we make a custom signer?

Reply:
I want to isolate Casper code, to make it easily replaceable in the future. I imagine we might have `Signer` trait and blockchain specific implementations for it.